.github/wiki/Project-Configurations.md

```diff
@@ -124,15 +124,5 @@ Only include the dSYM file for release builds.
 | Debug Production |  |
 | Release Production | ✔︎ |
 
-### Enable Bitcode
-
-Bitcode is a technology that enables recompiling applications to reduce their size. The recompilation happens when the application is uploaded to the App Store Connect or exported for Ad Hoc, Development, or Enterprise distribution.
-
-Enable build with bitcode for Production only.
-
-| Build configurations | Enable Bitcode |
-|---|---|
-| Debug Staging |  |
-| Release Staging |  |
-| Debug Production |  |
-| Release Production | ✔︎ |
+### Disable Bitcode
+- Acording to [Xcode 14 release notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes): Starting with Xcode 14, bitcode is no longer required for watchOS and tvOS applications, and the App Store no longer accepts bitcode submissions from Xcode 14.  The capability to build with bitcode will be removed in a future Xcode release. IPAs that contain bitcode will have the bitcode stripped before being submitted to the App Store.
```

Comment:
Too many spacing here 😃 
```suggestion
- Acording to [Xcode 14 release notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes): Starting with Xcode 14, bitcode is no longer required for watchOS and tvOS applications, and the App Store no longer accepts bitcode submissions from Xcode 14. The capability to build with bitcode will be removed in a future Xcode release. IPAs that contain bitcode will have the bitcode stripped before being submitted to the App Store.
```

Reply:
@blyscuit update the spacing in this [commit](https://github.com/nimblehq/ios-templates/pull/318/commits/1b9b2f2f36b64626b14d2b7bce7bda3438b78d57)

Reply:
@Shayokh144 @blyscuit From the release note, it said, "bitcode is no longer required for watchOS and tvOS applications". How about iOS applications?

Reply:
in documentation is is said "Starting with Xcode 14, bitcode is no longer required for watchOS and tvOS applications, and `the App Store no longer accepts bitcode submissions from Xcode 14.`"
If App Store no longer accepts bitcode submissions then it is good to disable that for all platforms.